org/wildfly/undertow/application-security-domain/capability.adoc

```diff
@@ -0,0 +1,26 @@
+NAME: org.wildfly.undertow.application-security-domain
+
+DESCRIPTION: Provides the mapping from a security domain as specified in a web application
+              to an HttpAuthenticationFactory plus additional policy information.
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: true
+
+SERVICE PROVIDED:
+
+   CLASS: java.util.function.BiFunction<DeploymentInfo, Function<String, RunAsIdentityMetaData>, Registration>
```

Comment:
Thanks for this. Very funky, but the key thing here is this is PRIVATE: true. No one outside this extension can count on this contract, so it can be whatever works best internally.